### ISSUE TITLE
Insufficient color contrast on highlighted code block in Dark Mode

### ISSUE THREAD

**Issue by pkrc267** [freeCodeCamp/freeCodeCamp#42405]
**Affected page**
[https://www.freecodecamp.org/learn/javascript-algorithms-and-data-structures/object-oriented-programming/define-a-constructor-function](https://www.freecodecamp.org/learn/javascript-algorithms-and-data-structures/object-oriented-programming/define-a-constructor-function)

**To Reproduce**

1. Go to [https://www.freecodecamp.org/learn/](https://www.freecodecamp.org/learn/). Login & switch to Dark mode from Menu.
2. Click on **Java Script Algorithm and Data Structure Certification**course
3. Scroll down to **'Object Oriented Programming'** module & open '**Define Constructor Function**' page.
4. Highlight the code block in the theory section by selecting & dragging with the mouse.

**Expected behavior**
On highlighting, the code becomes unreadable due to failing color contrast. This is a WCAG violation. There should be sufficient color contrast ratio (4.5:1) between text and background even when highlighted.

**Screenshots**
![image](https://user-images.githubusercontent.com/8735394/121166241-864bbb80-c86e-11eb-999c-ed3096817292.png)

**System:**
 - Device: Laptop
 - OS: Windows 10
 - Browser: Edge (chromium based)
 - Version: 91.0.864.41 (Official build) (64-bit)

**Comment by aknouche** [comment]
Hi @pkrc267 Can I work on this issue?

**Comment by gikf** [comment]
Hey @aknouche 

Issues are typically not assigned to anyone other than long-time contributors. You can read more about this at https://contribute.freecodecamp.org/#/FAQ?id=how-can-i-get-an-issue-assigned-to-me. 

Issues labelled with `help wanted` or `first timers only` are open for contributions.

Please make sure you read [our guidelines for contributing](https://contribute.freecodecamp.org/#/). Join us in [our chat room](https://chat.freecodecamp.org/channel/contributors) or [the forum](https://forum.freecodecamp.org/c/contributors/3) if you need help contributing, our moderators will guide you through this.

**Comment by aknouche** [comment]
Thanks for the reply @gikf, This issue is marked with the label "help wanted" so can I work on it and make a PR without being assigned to it then?
![github](https://user-images.githubusercontent.com/42774809/121205384-e98e1b80-c877-11eb-8dba-3d94d51d1997.PNG)

**Comment by ojeytonwilliams** [comment]
Yep, @aknouche, that's exactly right. You're very welcome to work on this.

**Comment by aknouche** [comment]
Ok will do thank you : )

**Comment by aknouche** [comment]
@ojeytonwilliams I have redproduced the issue. I have cloned the repo but I am having troubles finding the folder where the file for the lesson is located

**Comment by Nirajn2311** [comment]
The lesson doesn't matter here as the issue is present in every challenge which shows a code snippet. Basically a global css style has to be set just for the selection of the code snippet part.

**Comment by aknouche** [comment]
Hi, Ok thanks. I can see that this will be fixed through another issue so I
will let it be

Den ons 9 juni 2021 kl 06:23 skrev Niraj Nandish ***@***.***>:

**Comment by prsmahajan** [comment]
Hey, I want to work this issue, Can you assign me this?

**Comment by jeremylt** [comment]
@prsmahajan, it looks like there is already a PR open for this issue.
